.github/workflows/perf-models-impl.yaml

```diff
@@ -15,6 +15,9 @@ jobs:
           {name: "N300 WH B0", arch: wormhole_b0, runs-on: ["N300", "pipeline-perf", "bare-metal", "in-service"], machine-type: "bare_metal"},
         ]
         model-type: [llm_javelin, cnn_javelin, other]
+        include:
```

Comment:
Is this a keyword in YAML? I've never seen this before.

Reply:
Yeah this keyword came to my eyes and fortunately it works as I expected

Reply:
Why not just directly add to the above?

Reply:
Above is like combination tests, in this case 2 test-info and 3 model-type produces total 6 cases, so I added this as a separate test making it as 7(6+1)

Reply:
Sounds good

Please add a comment explaining this logic right above `include:`